app/src/main/java/ly/count/android/demo/App.java

```diff
@@ -296,10 +297,7 @@ public void onReceive(Context context, Intent intent) {
         };
         IntentFilter filter = new IntentFilter();
         filter.addAction(CountlyPush.SECURE_NOTIFICATION_BROADCAST);
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            registerReceiver(messageReceiver, filter, getPackageName() + COUNTLY_BROADCAST_PERMISSION_POSTFIX, null, Context.RECEIVER_VISIBLE_TO_INSTANT_APPS | Context.RECEIVER_NOT_EXPORTED);
-        } else {
-            registerReceiver(messageReceiver, filter, getPackageName() + COUNTLY_BROADCAST_PERMISSION_POSTFIX, null);
-        }
+
```

Comment:
would hinder if I wanna test api 25?

Reply:
nope, it will work fine. In some of the places they recommend to use ContextCompat for the receiver